snuba/cli/consumer.py

```diff
@@ -32,10 +33,16 @@
 @click.option(
     "--dataset",
     "dataset_name",
-    default="events",
     type=click.Choice(DATASET_NAMES),
     help="The dataset to target",
 )
+@click.option(
+    "--storage",
+    "storage_name",
+    default="events",
```

Comment:
Do we need a default here ? The default on the dataset was tech debt we never had time to clean up. Since we are migrating from using the dataset field to the storage I would just have it optional now, then when we remove the dataset we can make it required and pass it everywhere.

Reply:
For the meantime, we need it for backwards compatibility. This is good to know, when we switch over to use storage we can make sure it's always being passed and remove the default.